js/main.js

```diff
@@ -1,2 +1,119 @@
 /* global data */
 /* exported data */
+
+function createEntry(entry) {
+  var entriesListEl = document.querySelector('#entries-list');
+  var liEl = document.createElement('li');
+  var divImgEl = document.createElement('div');
+  var imgEl = document.createElement('img');
+  var divEl = document.createElement('div');
+  var ptitleEl = document.createElement('p');
+  var ptextEl = document.createElement('p');
+
+  liEl.setAttribute('class', 'row');
+  divImgEl.setAttribute('class', 'column-half');
+  imgEl.setAttribute('src', entry.url);
+  divEl.setAttribute('class', 'column-half');
+  ptitleEl.setAttribute('class', 'title');
+  ptitleEl.textContent = entry.title;
+  ptextEl.setAttribute('class', 'text');
+  ptextEl.textContent = entry.notes;
+
+  liEl.appendChild(divImgEl);
+  divImgEl.appendChild(imgEl);
+  liEl.appendChild(divEl);
+  divEl.appendChild(ptitleEl);
+  divEl.appendChild(ptextEl);
+  entriesListEl.prepend(liEl);
+  // entriesListEl.appendChild(liEl);
```

Comment:
You shouldn't be adding the element to the DOM inside of the function, you should `return` the DOM tree so that you can add it to the DOM where the function is called. That will allow you to only have one function to generate the DOM tree.